listenbrainz/stats/calculate.py

```diff
@@ -24,12 +27,26 @@ def calculate_user_stats():
 def calculate_stats():
     calculate_user_stats()
 
-if __name__ == '__main__':
+
+cli = click.Group()
+
+@cli.command()
+def calculate():
+    """ Command to calculate statistics from Google BigQuery.
+    This can be used from the manage.py file.
+    """
+
+    if not config.WRITE_TO_BIGQUERY:
+        time.sleep(10000)
```

Comment:
This should be in a loop, so it doesn't try to actually use BQ after 10000 seconds....